src/snaq_optimization.jl

```diff
@@ -1962,6 +1959,12 @@ Output:
 There are many optional arguments, including
 
 - `hmax` (default 1): maximum number of hybridizations allowed
+- `propQuartets` (default 1): the proportion of observed quartet concordance factors in `d`
+  to use when calculating network pseudo-likelihoods. Smaller values will lead to faster
+  method runtime but may come at the expense of accuracy if lowered too far.
+- `probQR` (default 0): the probability at any given step to use weighted random sampling
+  of quartets when deciding where to make topological moves when proposing the next
+  candidate network.
```

Comment:
`qtolAbs` and `qinfTest` are new optional arguments but have no documentation here